### ISSUE TITLE
wave animation cycle is a triangle wave

### ISSUE THREAD

**Issue by Wilker-a** [markusfisch/WaveLinesWallpaper#4]
the current wave animation cycle acts like a triangle wave, i.e when the waves finishes going up they immediately go down without variable speeds.

![wave-live-demo](https://user-images.githubusercontent.com/10854855/60936472-b83bf800-a2bc-11e9-8df9-2ee7c061f207.gif)

these are okay, but it would be good to open room for adding different types of animation cycles like sine waves and smoother speed changes.

**Comment by markusfisch** [comment]
You're right and I'm not completely happy with the current wave animation too. It's okay for some parameters but looks not so good for others. So I'll make some changes and add some more options too.

**Comment by Wilker-a** [comment]
another thing that can be added is an option to manually set the speed of the waves, be they horizontally or the animation cycle too, where  the slider could be a "range slider" or something looking like this —**•——•**– (sorry, idk what to call them)

**Comment by Wilker-a** [comment]
( i accidentally closed the issue D:)

**Comment by markusfisch** [comment]
Yes, I know what you mean 😉 While there's no such control in the Android SDK, there are a lot of open source components that offer that functionality.

Meanwhile, I've changed the wave calculation to use the sine function and it's a lot more smooth. Also I've added a control to change how fast the wave is oscillating.
